tasks/task452_emea_bg_el_classification.json

```diff
@@ -0,0 +1,1061 @@
+{
+    "Contributors": [
+        "Vijay Kumawat"
+    ],
+    "Source": [
+        "emea bg-el (dataset link- https://huggingface.co/datasets/emea)"
+    ],
+    "Categories": [
+        "Classification"
+    ],
+    "Input_language": "Bulgarian",
+    "Output_language": "Greek",
+    "instruction_language": "English",
+    "Definition": "In this task, you are given a sentence in the Bulgarian language and corresponding Greek translation of this sentence. Here, your job is to generate label \"True\" if translation is correct, otherwise generate label \"False\".",
+
+    "Positive Examples": [
+        {
+            "input": "Поддържащата доза е 15 mg веднъж дневно, но при някои пациенти по- високите дози може да са от полза.",
+            "output": "True",
```

Comment:
Where are two langauges ? I can see one 
You can use Input = Bulgraian : ...... , Lithuanian : .....